tests/features/support/steps.js

```diff
@@ -89,44 +89,51 @@ Given(/I start the interview ?(.*)/, async (optional_device) => {
 
 // Need to see if it's possible to remove the need for this on most occasions
 When(/I wait (\d+) seconds?/, async (seconds) => {
-  await scope.page.waitFor(seconds * 1000);
+  await scope.afterStep(scope, {waitForTimeout: (seconds * 1000)});
 });
 
 When("I do nothing", async () => {
   /* Here to make writing tests more comfortable. */
-  return true;  // Not sure this achieves anything
+  await scope.afterStep(scope);
 });
 
 Then('I should see the phrase {string}', async (phrase) => {
   /* In Chrome, this `innerText` gets only visible text */
   const bodyText = await scope.page.$eval('body', elem => elem.innerText);
   expect(bodyText).to.contain(phrase);
+
+  await scope.afterStep(scope);
```

Comment:
Looks like cucumber.js is adding in support for the `AfterStep` function (will run after each step). Once this is implemented, we can move this line into an `AfterStep` function instead of including it in each step function. See: https://github.com/cucumber/cucumber-js/pull/1416

Reply:
Holy moly! Awesome find! I must have looked for this 24 days ago because what I found then was years of comments on issues requesting exactly this feature. At least I hope it was before that. Otherwise I totally whiffed it.

We'll have to see if how we want to pass in the needs of the current step - whether it's waiting for `show if` or not, etc.